src/apps/Permissions/Home/Home.js

```diff
@@ -1,37 +1,54 @@
-import React from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
+import { Tabs } from '@aragon/ui'
 import { AppType } from '../../../prop-types'
-import BrowseByApp from './BrowseByApp'
-import BrowseByEntity from './BrowseByEntity'
-
-class Home extends React.Component {
-  static propTypes = {
-    apps: PropTypes.arrayOf(AppType).isRequired,
-    appsLoading: PropTypes.bool.isRequired,
-    permissionsLoading: PropTypes.bool.isRequired,
-    onOpenApp: PropTypes.func.isRequired,
-    onOpenEntity: PropTypes.func.isRequired,
-  }
-
-  render() {
-    const {
-      apps,
-      appsLoading,
-      permissionsLoading,
-      onOpenApp,
-      onOpenEntity,
-    } = this.props
-
-    return (
-      <React.Fragment>
-        <BrowseByApp apps={apps} loading={appsLoading} onOpenApp={onOpenApp} />
-        <BrowseByEntity
-          loading={appsLoading || permissionsLoading}
-          onOpenEntity={onOpenEntity}
-        />
-      </React.Fragment>
-    )
-  }
+import { usePermissionsByRole } from '../../../contexts/PermissionsContext'
+import Apps from './Apps'
+import AllPermissions from './AllPermissions'
+
+const TABS = ['App permissions', 'System permissions']
```

Comment:
We might need to move the tab handler one level up, when I enter an app from the `System permissions` and come back, the `App permissions` tab is selected

Reply:
Done https://github.com/aragon/aragon/pull/899/commits/16796f0ab35f02ab246648d4f517f746561df37a